--- libhandy-1.2.0/meson.build.orig	2021-03-12 13:10:27.000000000 +0100
+++ libhandy-1.2.0/meson.build	2021-04-01 20:30:07.815680135 +0200
@@ -103,13 +103,17 @@
 
 # Setup various paths that subdirectory meson.build files need
 package_subdir = get_option('package_subdir') # When used as subproject
-datadir        = get_option('datadir')  / package_subdir
-libdir         = get_option('libdir')  / package_subdir
-girdir         = get_option('datadir') / package_subdir / 'gir-1.0'
-typelibdir     = get_option('libdir')  / package_subdir / 'girepository-1.0'
 if package_subdir != ''
+  datadir        = get_option('datadir') / package_subdir
+  libdir         = get_option('libdir')  / package_subdir
+  girdir         = get_option('datadir') / package_subdir / 'gir-1.0'
+  typelibdir     = get_option('libdir')  / package_subdir / 'girepository-1.0'
   vapidir = get_option('datadir') / package_subdir / 'vapi'
 else
+  datadir        = get_option('datadir')
+  libdir         = get_option('libdir')
+  girdir         = get_option('datadir') / 'gir-1.0'
+  typelibdir     = get_option('libdir')  / 'girepository-1.0'
   vapidir = get_option('datadir') / 'vala' / 'vapi'
 endif
 
